torchdata/dataloader2/reading_service.py

```diff
@@ -252,10 +253,17 @@ def clean_me(process, req_queue, res_queue):
             # TODO(620): Make termination a function of QueueWrapperDataPipe (similar to reset)
             req_queue.put(communication.messages.TerminateRequest())
             _ = res_queue.get()
-            process.join()
+            process.join(20)
```

Comment:
Could you please add this constant to `_constant.py`?

Reply:
Putting this in a separate PR since the current one is almost landed.